### ISSUE TITLE
kubernetesVersion worked does not work with kind v0.11.1

### ISSUE THREAD

**Issue by grunty-tagme** [tilt-dev/ctlptl#109]
kind.ctlptl.yaml:
```
apiVersion: ctlptl.dev/v1alpha1
kind: Cluster
product: kind
registry: ctlptl-registry
kubernetesVersion: "v1.21.1"
---
apiVersion: ctlptl.dev/v1alpha1
kind: Registry
name: ctlptl-registry
```

```
ctlptl apply -f ./k8s/kind.ctlptl.yaml
Creating registry "ctlptl-registry"...
registry.ctlptl.dev/ctlptl-registry created
creating cluster: No available kindest/node versions for kind version v0.11.1.
Please file an issue: https://github.com/tilt-dev/ctlptl/issues/new
make: *** [cluster-up] Error 1
```

It worked with kind 0.9.0. If I remove the version, kind starts just fine:
```
ctlptl apply -f ./k8s/kind.ctlptl.yaml
registry.ctlptl.dev/ctlptl-registry created
Creating cluster "kind" ...
 ✓ Ensuring node image (kindest/node:v1.21.1) 🖼 
 ✓ Preparing nodes 📦  
 ✓ Writing configuration 📜 
 ✓ Starting control-plane 🕹️ 
 ✓ Installing CNI 🔌 
 ✓ Installing StorageClass 💾 
Set kubectl context to "kind-kind"
```

**Comment by eupestov** [comment]
Oh, I see new kind versions require the kindK8sNodeTable to be updated. I'll submit a PR shortly.

**Comment by nicks** [comment]
thanks for the fix! This should be fixed in v0.5.4
